src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -646,7 +646,8 @@ export default class ResultsViewPage extends React.Component<
                         )}
 
                     {(this.resultsViewPageStore.studies.isPending ||
-                        !tabsReady) && (
+                        (!tabsReady &&
```

Comment:
@onursumer i'm gonna merge this but we should probably figure out why tabsReady goes false here.